utils/zerofrom/derive/src/lib.rs

```diff
@@ -98,10 +98,16 @@ fn zf_derive_impl(input: &DeriveInput) -> TokenStream2 {
                 let binding = format!("__binding_{}", i);
                 let field = Ident::new(&binding, Span::call_site());
 
-                if has_clone_attr(&f.attrs) {
+                if has_attr(&f.attrs, "clone") {
                     quote! {
                         #field.clone()
                     }
+                } else if has_attr(&f.attrs, "copy") {
+                    let fty = f.ty.clone();
+                    zf_bounds.push(parse_quote!(#fty: Copy));
+                    quote! {
+                        *#field
```

Comment:
issue: instead of introducing a new feature we should always have it default to copying when there are no type params or lifetimes, and document this behavior ("This derive will require that all fields with lifetime parameters implement ZeroFrom, and all fields without them to implement Copy, with `#[zerofrom(clone)]` as an escape hatch")

Reply:
can we move this check to the branch below where the visitor is already called?

Reply:
certainly